packages/core/package.json

```diff
@@ -105,6 +105,7 @@
     "@types/emoji-mart": "^3.0.14",
     "@types/hast": "^3.0.0",
     "@types/uuid": "^8.3.4",
+    "@hocuspocus/provider": "^2.15.2",
```

Comment:
I think this is peerOptional no?
https://stackoverflow.com/questions/62047806/how-do-i-set-a-peer-dependency-optional#66228639

TS might care for someone who imports it